### PR TITLE
Tap to move through slides

### PR DESCRIPTION
**Pull request by theswerd**
## What does this PR accomplish?
This will make it so when a user taps on a slide, that it moves to the next one.

## Did you add any dependencies?

Nope

## How did you test the change?

iOS simulator

### CODE REVIEW

**Comment by patniemeyer** [comment]
@theswerd This is good but the elastic curve causes the card to do a little dance before it starts moving.  I suggest `curve: Curves.easeOut` maybe?  Also when you reach the last card we should ignore the tap (currently pushes the card off the screen briefly).  Other than those looks great!

**Comment by patniemeyer** [comment]
@theswerd Maybe `Curve.easeInOutCubic` is better... a bit faster.

**Comment by theswerd** [comment]
@patniemeyer done!

File: client/flutter/lib/carousel_page.dart [hunk 1]
```diff
@@ -63,6 +68,10 @@ class CarouselView extends StatelessWidget {
                 padding: EdgeInsets.only(bottom: 20),
                 child: pageViewIndicator(context)),
           ),
+          GestureDetector(
+          onTap: ()=>pageController.page == this.items.length-1?null:pageController.nextPage(duration: Duration(milliseconds: 500), curve: Curves.easeInOutCubic)
```
Comment (sstur) [inline client/flutter/lib/carousel_page.dart]: Nit: Is the spacing around operators here intentional? Should we incorporate dartfmt in our pipeline to standardize style?

Comment (theswerd) [reply client/flutter/lib/carousel_page.dart]: When I wrote this my IDE wasn't working, I will re push after formatting

Comment (theswerd) [reply client/flutter/lib/carousel_page.dart]: @sstur thx, just fixed

Comment (creativecreatorormaybenot) [reply client/flutter/lib/carousel_page.dart]: @sstur Check #362.

**Pull request merged**